include/Data/Vector.spec

```diff
@@ -7,18 +7,18 @@ data variance Data.Vector.Vector covariant
 
 measure vlen    :: forall a. (Data.Vector.Vector a) -> Int
 
-invariant       {v: Data.Vector.Vector a | 0 <= vlen v } 
+invariant       {v: Data.Vector.Vector a | 0 <= vlen v || 0 <= len v }
 
-assume !           :: forall a. x:(Data.Vector.Vector a) -> vec:{v:Nat | v < vlen x } -> a 
+assume !           :: forall a. x:(Data.Vector.Vector a) -> vec:{v:Nat | v < vlen x || v < len x } -> a
 
-assume unsafeIndex :: forall a. x:(Data.Vector.Vector a) -> vec:{v:Nat | v < vlen x } -> a 
+assume unsafeIndex :: forall a. x:(Data.Vector.Vector a) -> vec:{v:Nat | v < vlen x || v < len x } -> a
 
-assume fromList  :: forall a. x:[a] -> {v: Data.Vector.Vector a  | vlen v = len x }
+assume fromList  :: forall a. x:[a] -> {v: Data.Vector.Vector a  | vlen v = len x || len v = len x }
```

Comment:
Similarly, `fromList`, `length`, `replicate`, `imap` and `map`
should also have `&&` (instead of `II`) in the output.

Reply:
Thanks for advice!  :)